cmd/agent/app/builder.go

```diff
@@ -117,7 +117,7 @@ func (b *Builder) getMetricsFactory() (metrics.Factory, error) {
 	if b.metricsFactory != nil {
 		return b.metricsFactory, nil
 	}
-	return b.Metrics.CreateMetricsFactory("jaeger_agent")
+	return b.Metrics.CreateMetricsFactory("jaeger-agent")
```

Comment:
what's the reasoning here? Maybe this should be `b.Metrics.CreateMetricsFactory("jaeger").Namespace("agent",nil)`? That would be more consistent with the way metrics are constructed in the other binaries